src/adapters/adapter-constants.js

```diff
@@ -0,0 +1 @@
+../constants.js
```

Comment:
It has to do with webpack. constants.js winds up being included in the webpack and not as a separate file. Basically, everything from the adapter and its dependencies don't get processed by webpack, and when the adapter gets loaded, there is no constants.js file (it's merged into the build/gateway.js file). The simplest workaround seemed to be to add the adapter-constants.js and making it a symlink seemed to work. I think that the "ideal" solution would be to figure out how to redirect the require to grab the constants from the gateway.js webpack, I just haven't figure out how to do that. I'll file an issue so that we don't forget.

Reply:
I created #308